### ISSUE TITLE
On Windows, pylava found by linter-python, but no warnings/errors shown in Atom

### ISSUE THREAD

**Issue by Alexander-Serov** [pchomik/linter-python#55]
On Windows, I have pylava installed and have set the path to pylava in the linter-python package settings. Linter-python does not complain at startup that the path is not find, so it sees it OK. However, no warnings or errors are shown in the Atom editor when working with a .py file. If I just launch pylava from the command line on the same file, it gives the output correctly. It seems like the interface between pylava and Atom linter is broken. I have repeated the same installation steps on a Mac and there it works, but something on Windows is wrong. 

Let me know if I can provide more info.

**Comment by ferares** [comment]
Same issue on Windows here, I've enabled the debug console output and Pylava is in fact found and produces output but nothing is shown in the GUI

This seems like a dead project but just in case here's part of the console printouts:

```
>>> NEW ARGS <<<
-l,mccabe,pyflakes,pylint,pep8,pep257,--sort,E,W,D,-f,pep8, [...]\handlers.py
>>> END <<<
```
I've added the [...] in reality it shows the full path to the file correctly

```
>>> RAW OUTPUT <<<
handlers.py:10:1: D203 1 blank line required before class docstring (found 0) [pep257]
handlers.py:10:1: D205 1 blank line required between summary line and description (found 0) [pep257]
handlers.py:10:1: D212 Multi-line docstring summary should start at the first line [pep257]
handlers.py:10:1: D400 First line should end with a period (not 'r') [pep257]
handlers.py:19:1: D107 Missing docstring in __init__ [pep257]
handlers.py:1:0: C0111 Missing module docstring [pylint]
handlers.py:9:1: E302 expected 2 blank lines, found 1 [pep8]
>>> END <<<
```
